packages/amplify-category-function/src/provider-utils/awscloudformation/utils/constants.ts

```diff
@@ -17,6 +17,8 @@ export const ephemeralField = 'ephemeral';
 export const versionHash = 'latestPushedVersionHash';
 export const cfnTemplateSuffix = '-awscloudformation-template.json';
 
+export const lambdaPackageLimitInMB = 250;
```

Comment:
Not a huge deal at all but it would save at least one conversion if this value was stored in bytes like:
```suggestion
export const lambdaPackageLimitBytes = 250 * 1024 ** 2; // 250 MB
```